test/stubs/contentful/get-entries-keyword.json

```diff
@@ -0,0 +1,131 @@
+{
+  "sys": {
+    "type": "Array"
+  },
+  "total": 1,
+  "skip": 0,
+  "limit": 100,
+  "items": [
+    {
+      "sys": {
+        "space": {
+          "sys": {
+            "type": "Link",
+            "linkType": "Space",
+            "id": "owik07lyerdj"
+          }
+        },
+        "id": "3mMeUatw00GqQ0gAM86ICq",
+        "type": "Entry",
+        "createdAt": "2017-02-06T18:12:19.388Z",
+        "updatedAt": "2017-02-15T19:19:48.571Z",
+        "revision": 2,
+        "contentType": {
+          "sys": {
+            "type": "Link",
+            "linkType": "ContentType",
+            "id": "keyword"
+          }
+        },
+        "locale": "en-US"
+      },
+      "fields": {
+        "keyword": "bookbot",
+        "environment": "thor",
+        "campaign": {
+          "sys": {
+            "space": {
+              "sys": {
+                "type": "Link",
+                "linkType": "Space",
+                "id": "owik07lyerdj"
+              }
+            },
+            "id": "68Oy1FcaR2EiaMieicaoom",
+            "type": "Entry",
+            "createdAt": "2017-02-15T19:19:34.100Z",
+            "updatedAt": "2017-02-15T19:19:34.100Z",
+            "revision": 1,
+            "contentType": {
+              "sys": {
+                "type": "Link",
+                "linkType": "ContentType",
+                "id": "campaign"
+              }
+            },
+            "locale": "en-US"
+          },
+          "fields": {
+            "campaignId": "2299"
```

Comment:
Not a biggie -- but would it be possible to define 2299 as a `testCampaignId` somewhere so it's not hardcoded in our various tests? We do that Phoenix JS [here](https://github.com/DoSomething/phoenix-js/blob/master/test/helpers/test-helper.js#L33)